testing/src/cypress/e2e/02-institution/01-login-institution-page.cy.ts

```diff
@@ -11,7 +11,7 @@ const institutionHelperActions = new InstitutionHelperActions();
 const [URL, USERNAME, PASSWORD] =
   institutionHelperActions.getBaseUrlAndLoginCredentialsInstitution();
 
-describe("[Institution Login] - Login Page", () => {
```

Comment:
I think that naming the tests should follow the pattern: 
```
Component 1:
  verifies X;
  should display Y;
  can do something;
```
Like Cypress examples do:
![image](https://user-images.githubusercontent.com/78114138/205755814-27eb17c0-31e3-4b27-a243-4ee9755b760d.png)